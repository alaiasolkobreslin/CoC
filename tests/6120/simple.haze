
let f = lambda P:Type -> lambda Q:Type -> lambda g:P -> g in

((f Type) Type)