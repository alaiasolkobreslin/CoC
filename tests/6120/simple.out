simple.haze

Let: f beta reduced: (fun P:Type -> (fun Q:Type -> (fun g:P -> g)))
(fun g:Type -> g)
Term typed: (forall g:Type , Type)

